src/main/kotlin/com/terraformation/backend/accelerator/api/ProjectModulesController.kt

```diff
@@ -21,20 +21,32 @@ import org.springframework.web.bind.annotation.RequestMapping
 import org.springframework.web.bind.annotation.RestController
 
 @AcceleratorEndpoint
-@RequestMapping("/api/v1/modules")
+@RequestMapping("/api/v1/projects/{projectId}/modules")
 @RestController
 class ProjectModulesController(
     private val moduleStore: ModuleStore,
 ) {
   @ApiResponse200
   @ApiResponse404
-  @GetMapping("/projects/{projectId}")
+  @GetMapping
   @Operation(summary = "Gets modules for a project.")
   fun listModules(
       @PathVariable("projectId") projectId: ProjectId,
   ): GetProjectModulesResponsePayload {
     val models = moduleStore.fetchModulesForProject(projectId)
-    return GetProjectModulesResponsePayload(models.map { ProjectModule(it) })
+    return GetProjectModulesResponsePayload(models.map { model -> ProjectModule(model) })
+  }
+
+  @ApiResponse200
+  @ApiResponse404
+  @GetMapping("/{moduleId}")
+  @Operation(summary = "Gets one module for a project.")
+  fun getModule(
+      @PathVariable("projectId") projectId: ProjectId,
```

Comment:
No need for the name argument on the annotation here since the default is the parameter name. Doesn't hurt anything, just looks inconsistent since it's not present on the other parameter's annotation.

I know the existing code is inconsistent about specifying names; part of it is historical reasons (the server started off not using Spring Boot, and the other framework we used needed the explicit names) and part of it is just me being sloppy. At some point we should go clean it up across the code base.